lib/rubygems/sigstore/config.rb

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 The Sigstore Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'config'
+
+class SigStoreConfig
+    def initialize; end
+    def config
+        Config.setup do |config|
+            config.use_env = true
+            config.env_prefix = 'sigstore'
+            config.env_separator = '_'
+        end
+    settings_file = Config.setting_files(
+        File.expand_path('../../../../', __FILE__),
```

Comment:
is this a relative path to get back to the root of the project? feels a bit odd but maybe it's just a ruby thing.

Reply:
I know, this is quite ugly, its a relative path from the root of the gem